userspace/libscap/engine/test_input/CMakeLists.txt

```diff
@@ -1,8 +1,4 @@
 include_directories(${LIBSCAP_INCLUDE_DIRS} ../noop)
 add_library(scap_engine_test_input test_input.c)
 
-if(USE_BUNDLED_ZLIB AND NOT MINIMAL_BUILD)
-    add_dependencies(scap_engine_test_input zlib)
```

Comment:
If i remember correctly all of these were needed to fix weird compile issues with `-jX` flag and bundled zlib; basically, on zlib-less systems (ie: where zlib cannot be found in the system), running with `-jX` would break because some engines are built before zlib gets built (as it is an hard dep).